roles/online_docs/templates/mkdocs/docs/python.md

```diff
@@ -0,0 +1,91 @@
+
+Python has several different options of how to install and manage python packages - here is described the recommended way for our clusters.
+The steps are
+
+- [first creating the Python virtual environment](#Creating new Python virtual environment)
+- [then initialization of the environment](#To use the existing environment)
+- [installing custom package(s) inside this environment](#To install packages inside the python environment)
+- [deactivation of the virtual environment](#Deactivation of the virtual environment)
+- [and how to submit a slurm job that uses virtual python environment](#Using python virtual environemnt inside slurm job)
+
+
+## Creating new Python virtual environment
+
+To create a new Python virtual environment on the user interface machine (which can be later also used on compute nodes) use
+
+- Load latest version of Python
+  `module load Python`
+- Check the Python's version
+  `python3 --version`
+- Build a Python Virtual Environment
+  `python3 -m venv /groups/umcg-GROUP/TMPXX/MY_PYTHON_SUBDIR`
+  and replace the `GROUP`, `TMPXX` and `MY_PYTHON_SUBDIR` with appropriate values. Remember `tmpXX` folders are also available on compute nodes and therefore the compute jobs can access this python environment.
+
+
+## To use the existing environment
+
+- Source the environments 
+  `source /groups/umcg-GROUP/TMPXX/MY_PYTHON_SUBDIR/bin/activate`
+
+
+## To install packages inside the python environment
+
+- (Recommended) Upgrade `pip` and `wheel` packages, so that that you can install all the latest package versions
+  `pip install --upgrade pip wheel`
+- Install package
+  `pip install pypackage`
+  where `mypackage` is one or more package from the `https://pypi.org/`.
+
+
+# Deactivation of the virtual environment
+
+- to deactivate the virtual environment, simply run command `deactivate`
+
+
+## Using python virtual environemnt inside slurm job
+
+Create a slurm script with appropriate fields. For more information check also the documentation page about [batch jobs](analysis/#1-batch-jobs).
+
+```
+    #!/bin/bash
+    #SBATCH --job-name=pyvenv
+    #SBATCH --output=pyvenv-%N-%u-%j.out
+    #SBATCH --error=pyvenv-%N-%u-%j.err
+    #SBATCH --time=00:15:00
+    #SBATCH --cpus-per-task=1
+    #SBATCH --mem-per-cpu=1gb
+    #SBATCH --nodes=1
+    #SBATCH --open-mode=truncate
+    #SBATCH --export=NONE
+    #SBATCH --get-user-env=60L
+    
+    # clear all loaded modules
+    module purge
+    # to load the latest version of Python
+    module load Python
+    # initialize the python virtual environment
+    source /groups/umcg-GROUP/TMPXX/MY_PYTHON_SUBDIR/bin/activate
+    # check the python version (useful for debugging)
+    python3 --version
+    # print the location of python3 executable binary (also useful for debugging)
+    which python3
+    
+    # run the python command or in this case some custom python script
+    python3 ./my_python_script.py
+```
+
+Then submit this slurm SLURM job
+
+- `sbatch venv.slurm.sh`
```

Comment:
I usually try to keep the job-name in sync with the filename without extensions. E.g. if you have
```
#SBATCH --job-name=pyvenv
```
I would make the file ```pyvenv.sh``` and change the sbatch example to ```sbatch pyvenv.sh```

Reply:
```suggestion
- `sbatch pyvenv.sh`
```